images/tkg/docs/examples/prometheus_node_exporter.md

```diff
@@ -0,0 +1,96 @@
+# Running Prometheus node exporter service on the nodes
+
+## Use case
+
+I want to monitor the nodes using the hardware and OS metrics exposed by Prometheus [Node Exporter](https://prometheus.io/docs/guides/node-exporter/).
+
+## Customization
+
+For this we will create a new ansible task that will create and configure the node exporter service.
+
+- Create unit file for the new service `node_exporter.service` in [ansible files](./../../ansible/files/) folder.
```

Comment:
It would be better if we can add this as Step 1, 2, 3....

Reply:
As we have code blocks in between the points, and markdown linter complaints about the ordering(Like the second should've been `1.` instead of `2.`), I will keep bulleted points for now in all the documents 😐.